source/GameAction.cpp

```diff
@@ -127,6 +127,23 @@ namespace {
 			message += "flagship.";
 		Messages::Add(message, Messages::Importance::High);
 	}
+
+	double OutfitBulkBonus(int count)
+	{
+		if(count < 50)
+			return 1.05;
+
+		if(count < 100)
+			return 1.1;
+
+		if(count < 250)
+			return 1.25;
+
+		if(count < 500)
+			return 1.5;
+
+		return 2.;
```

Comment:
Do you want this to cap at 2.?
We can easily do smth like return 1. + count*0.01
And if you want the 2 capping then `return count > 500 ? 2. : 1. + count * 0.01`

Reply:
Also yeah should prob be defined in mission files eventually but that can be left for another PR if you want